apps/run-tests-in-parallel.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
Maybe worth a comment at the top of the file to orient newcomers?

Reply:
So I actually removed a comment from here in final PR review because the commend ended up being....

`apps/run-test-in-parallel.sh`
# Run our tests in parallel

Happy to add a comment if you still think it'd benefit, maybe you have an idea what's missing that I don't see because the context is too familiar to me atm?

Reply:
Oh funny.  TIL (made much more clear with the new name) this is literally what running tests invokes.  I'm not sure whether it's worth mentioning that this is invoked by the top level `yarn test` command, but a detail about who invokes this is the main thing I could think of.

Reply:
You got me thinking.... I think I'm gonna note a few things in the comment (e.g. that this === `yarn test`). Specificall unlike `npx karma start test` which will automatically invoke every test, if you add a new testType or levelType, you have to add it here to get run by default.

Reply:
BTW, I'm exploring using `karma-parallel` which would allow `npx karma start` to run every test without listing them explicitly and without needing this file.... but it seemed an easy thing to split into another PR and this one already sprawled 5x what I'd intended when I started fixing sourcemaps :laugh:

Reply:
Ironically far from lowering the memory, one of the key roles of the script formerly known as  `test-low-memory.sh` was to INCREASE the default node memory to permit our tests to run haha